shell.nix

```diff
@@ -0,0 +1,15 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+with pkgs;
+
+mkShell {
+  nativeBuildInputs = [
+    ncurses5
```

Comment:
won't you need `ncurses5` to run the program? 

`buildInputs` are all packages you need to run the program and `nativeBuildInputs` are all packages you need to build it